app/models/post.rb

```diff
@@ -12,4 +13,18 @@ class Post < ApplicationRecord
   validates :content, presence: true, length:
     {maximum: Settings.post.content_length}
   validates_associated :car
+
+  delegate :image, :price, to: :car, prefix: true
+  delegate :name, to: :user, prefix: true
+
+  delegate :year_of_manufacture_name, to: :car, prefix: true
+  delegate :gearbox_name, to: :car, prefix: true
+  delegate :origin_name, to: :car, prefix: true
+  delegate :car_type_name, to: :car, prefix: true
+  delegate :brand_name, to: :car, prefix: true
+  delegate :car_model_name, to: :car, prefix: true
+  delegate :color_name, to: :car, prefix: true
```

Comment:
ôi dời
cái nào chung model thì delegate chung 1 dòng thôi chứ e
  delegate :image, :price, :year_of_manufacture_name, :gearbox_name,  to: :car, prefix: true